Container/CSS/style.css

```diff
@@ -45,31 +52,32 @@
     gap: 1.2rem;
 }
 
-.nav-bar a{
-    padding: 0.4rem 1rem ;
+.nav-bar a {
```

Comment:
just beautify the file.i.e. remove extra spaces.

Reply:
resolved, thanks!